### ISSUE TITLE
About GPU memory！

### ISSUE THREAD

**Issue by BingY998** [mengmengliu1998/LAformer#2]
Hello, thank you very much for your excellent work！ I would like to ask if it is possible to reproduce this code if there is only one GTX3090 with 24G memory？

**Comment by mengmengliu1998** [comment]
In this code, the main role of the number of GPUs is to accelerate the training. only one GTX3090 with 24G memory is oke to train LAfomer, but maybe you need to reduce the batch size to one-half or one-quarter of the current and then increase the learning rate accordingly.

**Comment by BingY998** [comment]
Thank you very much. I will do it！
